task.go

```diff
@@ -257,3 +262,15 @@ func (task *Task) LastUpdated() time.Time {
 func (task *Task) UpdateStatus(status Status) {
 	task.Status = append(task.Status, status)
 }
+
+func (filter TaskFilter) Match(task *Task) bool {
+	if len(filter.Name) > 0 {
+		if filter.Name != task.Name {
+			return false
+		}
+	}
+	if filter.IsTerminated != task.IsTerminated() {
```

Comment:
As implemented this now always applies the isterminated filter so you can't ask for both terminated and non terminated at the same time. Even if we don't expose it in the API right away it would make sense to have the underlying implementation be a bit more flexible.

Reply:
yep, we can change query param from `terminated` to `state` and default query param from `terminated=false` to `state=terminated`

Reply:
yes, I like that much better than the boolean flag. I'm not sure exactly what to call it. status or state, terminated/nonterminated or terminal/nonterminal

thoughts @alde?

Reply:
Hmm, state=terminated I do like much better than terminated=[true/false]

Not so sure about nonterminated, it feels clunky and non-intuitive

How about it taking a comma-separated list?
?state=terminated,running,queued,... ?

Reply:
that could work but we would have to be careful to add new custom states liked QUEUED in the future. But because "nonterminated" is the default maybe we don't have to worry about that just now and the only query allowed is "state=terminated" and we still have the possibility of going to the comma list later on.

Reply:
Hi guys, I have't forget the PR at all. Will update it in upcoming days. 

I will group status in other "new" groups also `active` for `running/staging/starting/error` and `queued` for queued tasks. And if it is `state=all` then all tasks

WDYT?
I think it should be right to separate state from status as for now. As `status` query param in future could be used to directly reflect status like `failed/lost` etc. Only if needed.

Reply:
Yes, that sounds excellent. And good point about status vs. state

Reply:
Agreed, sounds great!